### PR TITLE
Feature/add ical export

### PR DESCRIPTION
**Pull request by drnlm**
This adds a simplistic ical export of the schedule.

### CODE REVIEW

File: wafer/schedule/admin.py [hunk 1]
```diff
@@ -221,6 +221,7 @@ def validate_schedule():
 class ScheduleItemAdminForm(forms.ModelForm):
     class Meta:
         model = ScheduleItem
+        readonly_fields = ('last_updated')
```
Comment (hodgestar) [inline wafer/schedule/admin.py]: Is this missing a trailing comma?

File: wafer/schedule/models.py [hunk 2]
```diff
@@ -261,6 +281,25 @@ def invalidate_check_schedule(*args, **kw):
...
+    for item in slot.scheduleitem_set.all():
+       item.save()
+    # We also need to update the next slot, in case we changed it's
+    # times as well
+    next_slot = slot.slot_set.all()
+    if next_slot.count():
+        # From the way we structure the slot tree, we know that
+        # there's only 1 next slot that could have changed.
+        for item in next_slot[0].scheduleitem_set.all():
+            item.save()
```
Comment (hodgestar) [inline wafer/schedule/models.py]: Are we sure that we can't trigger a save loop if two slots form a cycle?

File: wafer/schedule/admin.py (hunk 1 above)
Comment (drnlm) [reply wafer/schedule/admin.py]: Yes, it is

File: wafer/schedule/models.py (hunk 2 above)
Comment (drnlm) [reply wafer/schedule/models.py]: The validation on slots should ensure we can't have a slot cycle, but restricting these to only save the update_date is probably a good idea.

**Comment by drnlm** [comment]
Comments hopefully addressed.

**Pull request merged**